patches/tModLoader/Terraria/DataStructures/LegacyPlayerDrawLayer.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Terraria.ModLoader;
+
+namespace Terraria.DataStructures
+{
+	[Autoload(false)]
+	internal sealed class LegacyPlayerDrawLayer : PlayerDrawLayer
```

Comment:
Not sure what the reason of this is? Is it so players can use it? (If so, do we want that?)

Reply:
This class accepts delegates, and is used with vanilla draw methods. It's internal because we want to force modders to use OOP.

Reply:
So why isn't it called VanillaPlayerDrawLayer then ?